packages/testring-recorder-frontend/tsconfig.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
please, use `"extends": "../../tsconfig.json" instead of full copying content

Reply:
Done